projects/showcase/cypress/support/e2e-driver.ts

```diff
@@ -1,198 +1,112 @@
-      cy.get('mat-button-toggle').contains('countries names').click();
-      return this;
-    },
-    clickZoomToBoundsButton: (): E2eDriver => {
-      cy.get('.zoom-button').click();
-      return this;
-    },
-    clickHelloCustomButton: (): E2eDriver => {
-      cy.on('window:alert', this.alertStub);
-      cy.get('.custom-control').click();
-      return this;
-    },
-    clickEnableTerrainControlButton: (): E2eDriver => {
-      cy.get('.maplibregl-ctrl-terrain').click();
-      return this;
-    },
-    clickDisableTerrainControlButton: (): E2eDriver => {
-      cy.get('.maplibregl-ctrl-terrain-enabled').click();
-      return this;
-    },
-    clickHideControlsButton: (): E2eDriver => {
-      cy.get('button').contains('Hide Controls').click();
-      return this;
-    },
+    visitMapPage: (url: string) => this.helper.when.visit(url),
   };
 
-  assert = {
-    isSameAsSnapshot: (): E2eDriver => {
-      this.compareToReference().should('eq', 0);
-      return this;
-    },
-    isNotSameAsSnapshot: (): E2eDriver => {
-      this.compareToReference().should('be.gt', 0);
-      return this;
-    },
-    mapCanvasExists: (): E2eDriver => {
-      cy.get('canvas.maplibregl-canvas').should('have.length', 1);
-      return this;
-    },
-    mapObjectLoaded: (): E2eDriver => {
-      cy.get('[data-loaded="true"]').should('exist');
-      return this;
-    },
-    mapTerrainButtonIsOff: (): E2eDriver => {
-      cy.get('.maplibregl-ctrl-terrain').should('exist');
-      return this;
-    },
-    mapTerrainButtonIsOn: (): E2eDriver => {
-      cy.get('.maplibregl-ctrl-terrain-enabled').should('exist');
-      return this;
-    },
-    helloWorldPopupExists: (): E2eDriver => {
-      cy.get('.maplibregl-popup').should('exist');
-      cy.get('.custom-popup-class1').should('exist');
-      cy.get('.custom-popup-class2').should('exist');
-      cy.get('.maplibregl-popup-content').within(() => {
-        cy.get('div').should('have.text', 'Hello world !');
-      });
-      return this;
-    },
-    helloWorldPopupDoesNotExist: (): E2eDriver => {
-      cy.get('.maplibregl-popup').should('not.exist');
-      return this;
-    },
-    customHelloButtonExists: (): E2eDriver => {
-      cy.get('.custom-control').should('have.text', ' Hello ');
-      return this;
-    },
-    customHelloButtonDoesNotExist: (): E2eDriver => {
-      cy.get('.custom-control').should('not.exist');
-      return this;
-    },
-    fullscreenControlExists: (): E2eDriver => {
-      cy.get('.maplibregl-ctrl-fullscreen').should('exist');
-      return this;
-    },
-    fullscreenControlDoesNotExist: (): E2eDriver => {
-      cy.get('.maplibregl-ctrl-fullscreen').should('not.exist');
-      return this;
-    },
-    customPopupContainsHello: (): E2eDriver => {
-      cy.get('@alertStub').should('have.been.calledOnceWith', 'Hello');
-      return this;
-    },
-  };
+  get = {
+    canvas: (): Cypress.Chainable<HTMLCanvasElement> =>
```

Comment:
Is this used "outside" the driver? Seem like this should be internal...?

Reply:
Why internal? What if I want to assert things about the canvas in a test? That it exists? That it's visible?

Reply:
Do you do it right now? If not, let's make it private. We can always make it public in the future.